src/web3id.rs

```diff
@@ -0,0 +1,797 @@
+use std::{
+    collections::{BTreeSet, HashMap},
+    time::{SystemTime, UNIX_EPOCH},
+};
+
+use chrono::{DateTime, Utc};
```

Comment:
Good suggestion, will adapt 👍

Reply:
Will change this as `SystemTime` cannot represent time prior to unix epoch apparently.. Will use ISO8601 format as input instead which should provide consistent results.

Thanks for pointing this out as it was a potential pitfall.. I don't know why we use `contracts_common::Timestamp` inside `Web3IdAttribute::Timestamp`, as the former is described as not supporting values prior to unix epoch, while the outer interprets the inner timestamp as time since the minimum valid date in chrono (`"-26144-01-01T:00:00:00Z"`) which is quite a significant loss of precision 😅